akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -941,35 +940,69 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
   /**
    * Initiates a new round of gossip.
    */
-  def gossip(): Unit = {
-
+  def gossip(): Unit =
     if (!isSingletonCluster) {
-      val localGossip = latestGossip
+      val gossipTargets =
+        // TODO break this two methods out and unit test?
+        if (latestGossip.isMultiDc) multiDcGossipTargets()
+        else localDcGossipTargets()
+
+      selectRandomNode(gossipTargets) match {
```

Comment:
I think we should gossip less often to other DCs, so the round could start with a decision 80% local or 20% cross.